CMakeLists.txt

```diff
@@ -193,9 +193,9 @@ option(IREE_TARGET_BACKEND_DEFAULTS "Sets the default value for all compiler tar
 cmake_dependent_option(IREE_TARGET_BACKEND_VMVX "Enables the 'vmvx' compiler target backend" ON ${IREE_BUILD_COMPILER} OFF)
 
 # Supported default target backends.
-cmake_dependent_option(IREE_TARGET_BACKEND_DYLIB_LLVM_AOT "Enables the 'dylib-llvm-aot' compiler target backend" ${IREE_TARGET_BACKEND_DEFAULTS} ${IREE_BUILD_COMPILER} OFF)
+cmake_dependent_option(IREE_TARGET_BACKEND_LLVM_CPU "Enables the 'llvm-cpu' compiler target backend" ${IREE_TARGET_BACKEND_DEFAULTS} ${IREE_BUILD_COMPILER} OFF)
```

Comment:
```
CMake Error at build_tools/cmake/iree_check_test.cmake:245 (message):
  Unknown backend 'dylib-llvm-aot'.  Check IREE_TARGET_BACKEND_* options.

CMake Error at build_tools/cmake/iree_hal_cts_test_suite.cmake:62 (message):
  Unknown backend 'dylib-llvm-aot'.  Check IREE_TARGET_BACKEND_* options.
Call Stack (most recent call first):
  runtime/src/iree/hal/drivers/local_task/cts/CMakeLists.txt:15 (iree_hal_cts_test_suite)
```

Errrrr... I was hoping to make this change in a few steps, but these will need to be made together.